package Protocol;
message C2SRobotLoginReq
{
	required string verify_key = 1;
}